controller/responseTypeController.js

```diff
@@ -0,0 +1,39 @@
+const ResponseType = require('../model/responseTypeModel');
+const HttpStatus = require('http-status-codes');
+
+
+exports.getAll = function(req, res) {
+    
+    ResponseType.find().then( responseTypes => {
+        if(responseTypes) {
+            return res.status(HttpStatus.OK).json(responseTypes);
+        } else {
+            return res.status(HttpStatus.NOT_FOUND).json({
+                status: 'Error',
+                message: 'Not found.'
+            });
+        }
+    }).catch(err => {
+        return res.status(HttpStatus.INTERNAL_SERVER_ERROR);
+    });
+
+};
+
+exports.getByValue = function(req, res) {
+    const valueCode = req.params.id;
+
+    ResponseType.find({value: valueCode}).then( value => {
```

Comment:
use findOne when getting a single object from the database.

Reply:
find() returns an array as a result even when there is only one document that matches the query. That's the main reason why we should use findOne when querying for one document only.